railties/CHANGELOG.md

```diff
@@ -57,7 +57,7 @@
 
     *bogdanvlviv*
 
-*   Deprecate support of use `Rails::Application` subclass to start Rails server.
+*   Deprecate support for using `Rails::Application` subclass to start Rails server.
```

Comment:
How about “for using *a* `Rails::Application` subclass”?

Reply:
@georgeclaghorn Done!